SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -404,4 +404,63 @@
   <!-- End of StyleCop section-->
   <!-- **************************************************************************** -->
 
+
```

Comment:
New Roslyn-specific section containing two targets. The first target checks for the rule file and configures the build to use it. The second target runs after the compile phase and looks for the output report. If it finds it, it will set the appropriate SonarQube analysis setting.
Currently there is nothing that checks the version of MSBuild being used. However, this might not be necessary: for MSBuild 12 and earlier, the ErrorLog property is not used and the output file won't be generated, so the SonarQube analysis setting won't be set.